src/Datadog.Trace.ClrProfiler.Managed/ExtensionMethods/SpanExtensions.cs

```diff
@@ -7,6 +7,12 @@ internal static class SpanExtensions
         internal static string GetHttpMethod(this ISpan span)
             => span.GetTag(Tags.HttpMethod);
 
+        internal static string GetHost(this ISpan span)
```

Comment:
Where do  we _get_ tag values?

Reply:
I use these as convenience methods to grab tag values from the aspnetcore root span, for decorating the aspnetcoremvc span. Happy to remove if they don't make sense, as we're looking to rethink the nested mvc span anyways.

Reply:
Discussed offline.